docs/admin/kubernetes-ops/backup-restore.md

```diff
@@ -3,17 +3,38 @@ tags:
   - administration
   - kubernetes
   - operations
+  - backup
+  - restore
 ---
 
+The Jans Server should be backed up frequently, **we recommend once daily**.
 
-## This content is in progress
+There are multiple methods for backing up jans server. One way is manually using imperative commands. The other is automatically using open source tools.
 
-The Janssen Project documentation is currently in development. Topic pages are being created in order of broadest relevance, and this page is coming in the near future.
+## Manual Backup and Restore
 
-## Have questions in the meantime?
+Configmap backup:
+```bash
+kubectl get configmap -n <namespace> --field-selector metadata.name!=kube-root-ca.crt -o yaml > configmap-backup.yaml
+```
 
-While this documentation is in progress, you can ask questions through [GitHub Discussions](https://github.com/JanssenProject/jans/discussion) or the [community chat on Gitter](https://gitter.im/JanssenProject/Lobby). Any questions you have will help determine what information our documentation should cover.
+Secret backup:
+```bash
+kubectl get secret -n <namespace> -o yaml > secret-backup.yaml
```

Comment:
there is one secret that needs to get backed up only its normally called `cn` unless the user changed it in `values.yaml`